src/components/Panel/PanelExpansion.tsx

```diff
@@ -109,19 +106,15 @@ export class PanelExpansionItem extends Component<PanelExpansionProps> {
       consumerOnClick(index);
     } else {
       const openState = !this.state.openState;
-      const notControlled = open === undefined;
-      if (notControlled) {
-        this.setState({ openState });
-      }
+      this.setState({ openState });
       if (!!onClick) {
-        onClick({ openState: notControlled ? openState : !!open });
+        onClick({ openState });
       }
     }
   };
 
   render() {
     const {
-      open,
```

Comment:
The original issue had a mention "Make sure that implementation as prop is still needed internally for it to work." which was a bit strange wording. I understood it as "Note that implementation as prop is still needed internally for it to work.".

How did you understand this and can there be some repercussions for getting rid of the prop altogether?

Reply:
I understood it as to not removing everything related to `open`-prop, so that the opening/closing of the panel still works and the chevron icon still changes the direction to show the state.

Reply:
I'm sorry the instructions should have been more clear. This was my mistake.

Remove open-prop description only from styleguide. It is done by removing the `open` JSDOC comment from the interface.

If this now also removes part of the implementation, you can close without mergin and create new branch/PR.